composer/src/main/kotlin/com/gojuno/composer/Args.kt

```diff
@@ -129,8 +146,8 @@ fun parseArgs(rawArgs: Array<String>): Args {
                     }
                 },
                 verboseOutput = jCommanderArgs.verboseOutput ?: false,
-                devices = jCommanderArgs.devices ?: emptyList()
+                devices = jCommanderArgs.devices ?: emptyList(),
+                devicePattern = jCommanderArgs.devicePattern ?: ""
         )
-    }
+    }.let { args -> validateArguments(args) }
```

Comment:
ah because I didn't type this code in the IDE, you're able to use `this` inside of `apply {}`:

```kotlin
.apply { validateArguments(this) }
```

See https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/apply.html